src/stylesheets/layouts/_header.scss

```diff
@@ -2,44 +2,47 @@
   background-color: $color-mint;
   display: flex;
   justify-content: center;
+  position: relative;
```

Comment:
what is being positioned relative to this box?

Reply:
absolutely NOTHING